tools/bazel/vendor.bzl

```diff
@@ -54,6 +75,13 @@ def _log_cargo_vendor(repository_ctx, result):
 vendor = repository_rule(
     doc = "A rule used to vendor the dependencies of a crate in the current workspace",
     attrs = {
+        "cargo_version": attr.string(
```

Comment:
I think you should also add `cargo_iso_version` to complete the api of `load_arbitrary_tool`.